docs/help-center/tim-features/use-hba.mdx

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: Use HBA
+title: Use HBA
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Use HBA</h2>
+  </div>
+</div>
+
+## Register HBA card
+
+To connect to VZD Fhir Directory you have to add your HBA. This needs to be done with
```

Comment:
I would like to explain this differently. Most users will not know what exactly VZD Directory is. I suggest to write something like: If you have an eHBA and you would like to allow people from outside your organisation to be able to contact you, you can register on the TIM contact list by connecting your eHBA to your user account.
You can see here in the last slides (German) how I explained the new TIM Features to users who do not know anything about it yet. (Voice over in the notes field) https://docs.google.com/presentation/d/1HuoxQoX6p_kSfTnEQgQtesJXMzBlliJB/edit#slide=id.g2cd37064a1a_1_244